tests/unit/test_kwargs/test_path_params.py

```diff
@@ -60,7 +59,7 @@
                 "user_id": "abc",
                 "order_id": str(uuid1()),
             },
-            True,
+            False,
```

Comment:
Why? Sounds like if this raised before it should still raise.

Reply:
no, msgspec is not capable of validating uuids based on pydantic annotated metadata. pydantic v2 does not have a specific uuid type, it has annotated meta data for this.